command-expansion/tsconfig.json

```diff
@@ -0,0 +1,101 @@
+{
+  "compilerOptions": {
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+    /* Projects */
+    // "incremental": true,                              /* Enable incremental compilation */
+    // "composite": true,                                /* Enable constraints that allow a TypeScript project to be used with project references. */
+    // "tsBuildInfoFile": "./",                          /* Specify the folder for .tsbuildinfo incremental compilation files. */
+    // "disableSourceOfProjectReferenceRedirect": true,  /* Disable preferring source files instead of declaration files when referencing composite projects */
+    // "disableSolutionSearching": true,                 /* Opt a project out of multi-project reference checking when editing. */
+    // "disableReferencedProjectLoad": true,             /* Reduce the number of projects loaded automatically by TypeScript. */
+
+    /* Language and Environment */
+    "target": "es2016",                                  /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
```

Comment:
We should have this be ESNext to track any JS api updates - since we're pulling the latest node version

Reply:
Note we are using the LTS Node version for all our Docker base images (v16.4.0).

Reply:
@dyst5422 @camargo Will `ESNext` use the LTS version or the latest version which could be something higher in the future?

Reply:
It tracks the latest advancements to the language (that have been stabilized)

`Additional APIs available in ESNext - This changes as the JavaScript specification evolves`

Reply:
So I believe `ESNext` currently points to `ES2022`: https://www.typescriptlang.org/tsconfig#lib
According to this site https://node.green/#ES2022 Node LTS for ES2022 is ~70% feature complete.
Where `ES2021` is 100% feature complete: https://node.green/#ES2021 so I'd go with that

Reply:
Sounds good